sdk/iothub/Microsoft.Azure.Management.IotHub/src/generate.ps1

```diff
@@ -4,7 +4,7 @@
 
 try
 {
-	Start-AutoRestCodeGeneration -ResourceProvider "iothub/resource-manager" -AutoRestVersion "v2" -ConfigFileTag "package-2020-03"
```

Comment:
is omitting file tag intentional ?

Reply:
Yes, the tag was originally specified when we didn't want the default (latest). Now we do, so I've removed it.